### PR TITLE
added changes for v1.0.0

### PR DESCRIPTION
**Pull request by kavaribes**
### 1.0.0
* Updated Node to 14
* Request module replaced by axios
* Added deploy action for deploying k8s service and deployment to the cluster from GitHub
* Updated kubectl to v1.23.0

### CODE REVIEW

File: bin/ci/deployment-v2.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        git.owner: GITHUB_ORG
+        git.branch: GITHUB_BRANCH
+      annotations:
+        container.apparmor.security.beta.kubernetes.io/sftp: runtime/default
+    spec:
+      serviceAccountName: "CLUSTER_SERVICEACCOUNT_VALUE"
+      containers:
+        - name: sftp
+          #image: "gcr.io/ica-miami/docker-sftp:cf7ae7253694d7f95dd2cff0757d95b5ea0b489f"
+          image: "gcr.io/ica-miami/docker-sftp:IMAGE_VERSION"
```
Comment (planv) [inline bin/ci/deployment-v2.yml:32]: Would be good to move this to variables

File: bin/server.js [hunk 2]
```diff
@@ -97,110 +169,117 @@ function appEndpoint( req, res ) {
...
+    }
+
+    if (process.env.SLACK_NOTIFICACTION_URL && process.env.SLACK_NOTIFICACTION_URL.indexOf("https") === 0) {
+
+        axios({
+            method: 'post', //you can set what request you want to be
+            url: process.env.SLACK_NOTIFICACTION_URL,
+            data: {
+                channel: process.env.SLACK_NOTIFICACTION_CHANNEL,
+                username: 'Icam/SSH',
```
Comment (planv) [inline bin/server.js:276]: Username can be moved to Variables

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        sed -i.bak "s|GITHUB_ORG|$GITHUB_REPOSITORY_OWNER|g" bin/ci/deployment-v2.yml
+    - run: |
+        sed -i.bak "s|GITHUB_BRANCH|$GITHUB_REF_NAME|g" bin/ci/service.yml
+    - run: |
+        sed -i.bak "s|GITHUB_BRANCH|$GITHUB_REF_NAME|g" bin/ci/deployment-v2.yml
+
+    # Deploy the Docker image to the GKE cluster
+    - run: |
+        kubectl apply -n icamiami-org -f bin/ci/service.yml && \
+        kubectl apply -n icamiami-org -f bin/ci/deployment-v2.yml
```
Comment (planv) [inline .github/workflows/deploy.yml:106]: to vars

File: package.json [hunk 4]
```diff
@@ -1,36 +1,37 @@
...
-    "express": "^4.16.3",
-    "firebase-admin": "^5.13.1",
-    "lodash": "^4.17.10",
-    "mustache": "^2.3.0",
-    "newrelic": "^4.5.1",
-    "requestretry": "^1.13.0"
-  }
-}
+    "name": "rabbit-ssh",
+    "version": "0.1.4",
```
Comment (planv) [inline package.json:3]: Bump version

**Review by fqjony — APPROVED** [review]

**Pull request closed without merging**